### DISCUSSION TITLE
How Gracefully uninstall Cockpit?

### DISCUSSION THREAD

**Discussion by SlowFox72** [cockpit-project/cockpit#20593] [General]
What is the supported way to uninstall cockpit on ubuntu 22.04.4?
The following is that the correct and supported method?
`sudo apt remove cockpit`
`sudo apt purge cockpit`

Regards

Z

**Comment by SlowFox72** [comment]
I did the following in addtion to 
`sudo apt remove cockpit`
`sudo apt purge cockpit`

`sudo apt remove cockpit-bridge cockpit-networkmanager cockpit-packagekit  cockpit-pcp cockpit-storaged cockpit-system cockpit-ws`
`sudo apt purge cockpit-bridge cockpit-networkmanager cockpit-packagekit  cockpit-pcp cockpit-storaged cockpit-system cockpit-ws`

**Reply by martinpitt** [reply]
The short way to do this is `sudo apt purge cockpit-bridge cockpit-ws`, that'll take everything with it. You can check afterwards with `dpkg -l | grep cockpit`.

**Reply by SlowFox72** [reply]
Thank you

Good info
